kubeadm/flannel/Dockerfile.nanoserver

```diff
@@ -0,0 +1,35 @@
+ARG tag="nanoserver-1809"
```

Comment:
Moving this to nano is a good improvement. Is there any reason to keep the windowsservercore image?

Reply:
I don't know. Maybe not. Nano is faster and smaller.
Do you think we need to remove servercore?

Reply:
I removed servercore from build